packages/node_modules/@ciscospark/plugin-phone/src/call.js

```diff
@@ -949,6 +949,9 @@ const Call = SparkPlugin.extend({
   },
 
   _updateMedia(payload) {
+    if (payload.sdp && !payload.sdp.includes(`b=`)) {
```

Comment:
bandwidth. basically linus screws up if we don't include a bandwidth line in the sdp. this check is here to make sure we're always using the bandwidth limited sdp rather than the sdp read from peer.localDescription.sdp.